tigerpath/majors_and_certificates/majors/HIS.json

```diff
@@ -95,285 +100,606 @@
-                "EAS 336/HIS 319",
-                "EAS 320/HIS 320",
-                "EAS 321/HIS 321",
-                "REL 378/HIS 331",
-                "EAS 359/HIS 333",
-                "NES 337/HIS 334",
-                "EAS 361/HIS 335",
-                "NES 437/HIS 337",
-                "NES 325/HIS 338",
-                "EAS 340/HIS 340",
-                "NES 338/HIS 349",
-                "JDS 355/HIS 356",
-                "NES 394/HIS 409",
-                "NES 410/HIS 410",
-                "EAS 415/HIS 415",
-                "EAS 416/HIS 416",
-                "NES 433/HIS 433",
-                "NES 411/HIS 457",
-                "LAS 374/HIS 486"
-              ]
-            },
-            {
-              "name":"Pre-Modern",
-              "max_counted":1,
-              "min_needed":1,
-              "explanation":"Of the departmental courses, one must be a course in premodern history.",
-              "completed_by_semester":8,
-              "course_list":[
-                "HIS 207",
-                "HIS 210",
-                "HIS 211",
-                "HIS 290",
-                "HIS 291",
-                "HIS 294",
-                "HIS 303",
-                "HIS 307",
-                "HIS 314",
-                "HIS 324",
-                "HIS 343",
-                "HIS 344",
-                "HIS 345",
-                "HIS 347",
-                "HIS 355",
-                "HIS 358",
-                "HIS 367",
-                "HIS 368",
-                "HIS 369",
-                "HIS 371",
-                "HIS 407",
-                "HIS 413",
-                "HIS 421",
-                "HIS 422",
-                "HIS 428",
-                "HIS 435",
-                "HIS 437/HLS 437",
-                "HIS 452",
-                "HIS 476",
-                "EAS 218/HIS 209",
-                "CLA 220/HIS 215",
-                "CLA 216/HIS 216",
-                "CLA 217/HIS 217",
-                "CLA 218/HIS 218",
-                "CLA 219/HIS 219",
-                "NES 220/HIS 220",
-                "NES 390/HIS 221",
-                "HUM 470/HIS 230",
-                "HUM 470/HIS 301",
-                "NES 251/HIS 251",
-                "EAS 280/HIS 279",
-                "EAS 335/HIS 318",
-                "EAS 336/HIS 319",
-                "EAS 320/HIS 320",
-                "EAS 321/HIS 321",
-                "CLA 326/HIS 326",
-                "CLA 327/HIS 327",
-                "CLA 324/HIS 328",
-                "CLA 325/HIS 329",
-                "NES 437/HIS 337",
-                "EAS 340/HIS 340",
-                "REL 350/HIS 353",
-                "EAS 415/HIS 415",
-                "EAS 416/HIS 416",
-                "HLS 423/HIS 423",
-                "JDS 443/HIS 443",
-                "HUM 322/HIS 447",
-                "CHV 466/HIS 466"
+              "req_list": [
+                {
+                  "name":"Thematic Distribution Requirement",

```

Comment:
https://history.princeton.edu/academics/undergraduate/current-concentrators#DistributionReqs

"
Table of Contents:
Program Overview »
Prerequisites »
Distribution Requirements »
The Rule of 12 »
History of Science, Technology, and Medicine »
Cognate Approval »
Study Abroad »
Senior Departmental Examination »
Graduation Requirements and Honors »
Grading Practices »
Academic Integrity »

 

 

History Concentrators

 

Program Overview
Dickinson sundialThe undergraduate history program is designed to encourage students to immerse themselves in a particular time or place (depth) as well as to gain a broad sense of history (breadth). This is accomplished through both the independent research conducted by juniors and seniors and the distribution requirements that students must take. In their junior year, concentrators write two junior papers, one in each semester. In their senior year, students complete year-long independent research that culminates in the senior thesis.

Concentrators are encouraged to acquire a broad body of knowledge. All students must take four courses that satisfy the department’s thematic distribution requirements, and two that fulfill the geographical distribution requirement. (Thematic and geographical courses may overlap). In addition, students are encouraged to participate in study abroad programs and to master at least one language in addition to English. Knowledge of a foreign language is virtually essential for senior thesis research on many topics.

History concentrators must take ten courses in the Department: one 200-level prerequisite, the Junior Seminar (History 400), and eight other departmental courses. History courses taken in the freshman and sophomore years are counted among the ten required for graduation. Students may not designate any departmental courses “Pass-Fail” even if the minimum ten courses have been completed. Students should speak with the Director of Undergraduate Studies if they have questions about satisfying the prerequisite or potentially designating cognate courses as departmentals.


Prerequisites
Before they enter the department, students in the class of 2023 are required to take and pass one (1) 200-level course, from the list designated as prerequisites.

Prospective concentrators who have not fulfilled the 200-level prerequisite should consult with the Director of Undergraduate Studies and plan to take one of the appropriate courses in the fall of the junior year.


Distribution Requirements
Class of 2021 and Class of 2022 will continue with the existing requirements. Current concentrators who wish to migrate to the new requirements should consult with the Director of Undergraduate Studies.
 
"New requirements for the Class of 2023*
*Students who entered the concentration in the spring of 2020, took a gap year, and joined the class of 2023 should consult with the Director of Undergraduate Studies about the new requirements."